great-idea/css/index.css

```diff
@@ -97,15 +99,20 @@ h1 {
     padding: 5px 50px;
 
 }
+.features-about{
+    display: flex;
+}
+.services-vision{
+    display:flex;
+    justify-content: space-between;
+}
 .features {
-    display: inline-block;
-    width: 450px;
-    margin: 20px 20px 20px 0px;
+    display: flex;
```

Comment:
If you use `display: flex;` here, remember the default `flex-direction` is `row`. But notice that the elements are supposed to be in a column in the design file.

Before I say, "Therefore, put `flex-direction: column;`, it's important to understand that the placement of the `h2` and `p` are naturally in the right place. That is, the default flow of how the elements are placed on the page already fits the design file. In essence, there's no need to make `.features` a flex container. **Use flex and other positioning tools when you need to position elements outside the default flow of the page.**

In addition, `flex` doesn't solve all your problems: you still would need the margin and width properties you had previously to make this look more consistent with the design.